src/blinding.cpp

```diff
@@ -456,6 +456,27 @@ ustring blind15_sign(ustring_view ed25519_sk, std::string_view server_pk_in, ust
     return result;
 }
 
+ustring blind_version_sign_request(
+        ustring_view ed25519_sk,
+        uint64_t timestamp,
+        ustring_view method,
+        ustring_view path,
+        std::optional<ustring_view> body) {
+    auto [pk, sk] = blind_version_key_pair(ed25519_sk);
+
+    // Signature should be on `TIMESTAMP || METHOD || PATH || BODY`
+    ustring buf;
+    buf.reserve(10 + 6 + path.size() + (body ? body->size() : 0));
```

Comment:
should include method.size() too right?

Reply:
Done.  Also had forgotten to run `clang-format`, so did that too.